src/SFML.Window/WindowBase.cs

```diff
@@ -783,6 +783,15 @@ private void CallEventHandler(Event e)
 
         [DllImport(CSFML.window, CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
         private static extern bool sfWindowBase_createVulkanSurface(IntPtr CPointer, IntPtr vkInstance, out IntPtr surface, IntPtr vkAllocator);
+
+        [DllImport(CSFML.window, CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
+        private static extern Vector2i sfMouse_getPosition(IntPtr CPointer);
+
+        [DllImport(CSFML.window, CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
+        private static extern void sfMouse_setPosition(Vector2i position, IntPtr CPointer);
+
+        [DllImport(CSFML.window, CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
+        private static extern Vector2i sfTouch_getPosition(uint Finger, IntPtr RelativeTo);
         #endregion
```

Comment:
I think these should instead be `sfMouse_getPositionWindowBase`, `sfMouse_setPositionWindowBase` and `sfTouch_getPositionWindowBase`.

Reply:
Ehhrm, right 😅

Reply:
![PenguinHideGIF](https://github.com/SFML/SFML.Net/assets/920861/8c3c45cd-6d81-4870-b011-1ce5dc928699)